src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivitiesResource.kt

```diff
@@ -434,6 +435,48 @@ class ActivitiesResource(
   ): Page<FindActiveAllocationIdsResponse> =
     allocationService.findActiveAllocations(pageRequest, prisonId, excludeProgramCodes, courseActivityId)
 
+  @PreAuthorize("hasRole('ROLE_NOMIS_ACTIVITIES')")
+  @GetMapping("/allocations/suspended")
```

Comment:
The main change is to add an endpoint to find all prisoners on suspended allocations where the allocation is eligible for migrating to DPS. This is required when onboarding a new prison as manual effort is needed to deal with the suspensions so we will provide all the suspensions in the migration preview page.